tests/conftest.py

```diff
@@ -436,25 +436,27 @@ def dag_maker(request):
     DEFAULT_DATE = timezone.datetime(2016, 1, 1)
 
     class DagFactory:
-        def __enter__(self):
+        @provide_session
+        def __enter__(self, session=None):
             self.dag.__enter__()
+            self.session = session
```

Comment:
If we're going to store a session, we should do it in `__call__`, not `__enter__`

Reply:
Now used it on `__call__`